code/style.css

```diff
@@ -1,4 +1,253 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background-color: #002233;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 20px;
 }
+
+
+/*** MOBILE MAIN DESIGN ***/
+
+@media (min-width: 1px) and (max-width: 667px) {
+
+  /*** HEADER ***/
+
+  header{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    }
+    
+  nav {
+      display:none;
+    }
+
+  .order-1{
+      order: 1;
+    }
+    
+  .order-2 {
+      order: 2;
+    }
+  
+  .order-3 {
+      order: 3;
+    }
+  
+  .order-4 {
+      order: 4;
+    }
+    
+  a:link {
+      text-decoration: none;
+      color: white;
+      }
+    
+  a:visited {
+      text-decoration: none;
+      color: white;
+    }
+    
+  a:hover {
+      color: white;
+      text-decoration: underline;
+    }
+    
+  .logo {
+    width: 100px;
+    height: 100px;
+    margin: 20px;
+    }
+
+    /*** BIG NEWS ***/
+
+  .big-news {
+      padding: 30px;
+      margin-top: 30px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+    }
+
+    .big-news-text {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+    }
+  
+    /*** NEWS GRID ***/
+
+
+    .other-news {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+      
+    }
+      
+    .news1, .news2, .news3, .news4, .news5, .news6, .news7, .news8 {
+      display:flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+      background-color: #00334d;
+      margin: 10px;
+    }
+  
+}
+
+
+/*** TABLET RESPONSIVE ***/
+
+@media (min-width: 668px) and (max-width: 1024px) {
```

Comment:
If you have the mobile first styling as default, you don't need to specify all of it again in the next media query. A lot of the styling in the media queries are the same as for mobile. So - you only need to specify what's new/different for the media queries if that makes sense!